packages/cloud_firestore/cloud_firestore/pubspec.yaml

```diff
@@ -22,7 +22,7 @@ dependencies:
   flutter:
     sdk: flutter
   meta: ^1.0.5
-  firebase_core: ^0.4.0
+  firebase_core: ^0.4.4
```

Comment:
`Because cloud_firestore depends on firebase_core ^0.4.4 which doesn't match any versions,`

https://pub.dev/packages/firebase_core/versions/0.4.3+3

Is there a pending PR that needs to be merged/published before this one, @cbenhagen?

Reply:
Yeah, https://github.com/FirebaseExtended/flutterfire/pull/1704 which has already landed, but haven't been able to publish yet (permission issues)

Reply:
Done!

Reply:
I'll restart the checks in this PR then! Thanks @franciscojma86!